dashboard/app/models/stage.rb

```diff
@@ -61,6 +61,12 @@ def unplugged?
     script_levels.first.oldest_active_level.unplugged?
   end
 
+  def display_as_unplugged
+    script_levels = script.script_levels.select {|sl| sl.stage_id == id}
+    return false unless script_levels.first
+    script_levels.first.oldest_active_level.properties["display_as_unplugged"] == "true" || unplugged?
```

Comment:
We should probably add a note here that this only really works for CSF where we have unplugged levels. There are unplugged lessons in CSD and CSF but they don't use this set up so if we ever want this for those courses we will need to reconsider this

Reply:
How do CSD and CSP handle unplugged status of a lesson?